src/extension.ts

```diff
@@ -85,7 +85,7 @@ export function activate(context: ExtensionContext) {
    * @param document the text document.
    * @param languageId the language ID.
    */
-  async function tryToForceLanguageId(document: TextDocument, fileName: string, propertiesLanguageMismatch: PropertiesLanguageMismatch, languageId: string) {
+  async function tryToForceLanguageId(document: TextDocument, fileName: string, propertiesLanguageMismatch: PropertiesLanguageMismatch, languageId: string, userInitiated: boolean) {
```

Comment:
Not a problem with this PR, but we should remove the `async` modifier since this function does not use `await` anywhere in the body.

Reply:
Okay. I think the original intent may have been to show it returns a promise, so I added a `Promise<void>` return annotation.